src/components/DepositWidget/Row.tsx

```diff
@@ -1,7 +1,14 @@
 import React from 'react'
 import styled from 'styled-components'
 
-const WrapperRow = styled.tr``
+import { TokenBalanceDetails } from 'types'
+
+const WrapperRow = styled.tr`
```

Comment:
Can we fix the overflow of the button with longer token names such as CUSD?

Reply:
Yes, I was planning to review the styles a bit for the widget, but's not part of this PR